packages/neuron-ui/src/services/subjects.ts

```diff
@@ -67,7 +67,7 @@ export const Navigation = SubjectConstructor<Subject.URL>('navigation')
 export const SetLocale = SubjectConstructor<(typeof LOCALES)[number]>('set-locale')
 export const DeviceSignIndex = SubjectConstructor<Subject.SignIndex>('device-sign-index')
 export const MultisigOutputUpdate = SubjectConstructor<string>('multisig-output-update')
-export const Migrate = SubjectConstructor<'need-migrate' | 'migrating' | 'failed' | 'finish'>('migrate')
+export const Migrate = SubjectConstructor<'need-migrate' | 'migrating' | 'failed' | 'finish'>('migrate', true)
```

Comment:
What is `true` here for

Reply:
When calling unsubscribe the channel, it will only unsubscribe created by itself, but not unsubscribe all of the channel that name is the same.

Reply:
Got it